libs/utils/android/benchmark.py

```diff
@@ -203,9 +211,15 @@ def __init__(self):
             raise
 
         self._log.info('=== Execution...')
-        self.wl.run(out_dir=self.out_dir,
-                    collect=self._getBmCollect(),
-                    **self.bm_params)
+        for iter in range(self.bm_iterations):
```

Comment:
Right. Actually also waiting some time between iterations might be something people might want to configure. We could set reasonable defaults and add two more command line options?